README.md

```diff
@@ -34,133 +32,74 @@ Composer support has been added,  Place the following lines in your composer.jso
+Please follow the [installation procedure](#installation--usage) and then run the following:
 
 ```php
-$demo_district = CleverDistrict::retrieve("4fd43cc56d11340000000005");
-$demo_schools = $demo_district->schools();
-$demo_teachers = $demo_district->teachers();
-$demo_students = $demo_district->students();
-$demo_sections = $demo_district->sections();
-```
-
-The same patterns apply to other Clever objects (CleverSchool, CleverStudent, CleverSection, CleverTeacher, CleverSchoolAdmin, and CleverEvent). Here's an example that gets information about a teacher, all the sections they teach, the school they belong to, and the students they teach:
+<?php
+require_once('./autoload.php');
 
-```php
-$teacher = CleverTeacher::retrieve("4fee004dca2e43cf270007d5");
-$teacher_sections = $teacher->sections();
-$teacher_school = $teacher->school();
-$teacher_students = $teacher->students();
-```
+// Note: This is hard coded for demo purposes only. Keep your access tokens secret!
+// https://dev.clever.com/docs/security#section-security-best-practices
+Clever\Configuration::getDefaultConfiguration()->setAccessToken('TEST_TOKEN');
 
-## Query Parameters
+$api_instance = new Clever\Api\DataApi();
 
-Most API requests allow query parameters for purposes like paging and filtering. The same can be done in the PHP library by passing the `all()` method an associative array of parameters. For example, to retrieve the last Event from the Events API, you could run:
+try {
+    $result = $api_instance->getStudents();
+    print_r($result);
+} catch (Exception $e) {
+    echo 'Exception when calling DataApi->getStudents: ', $e->getMessage(), PHP_EOL;
+}
 
-```php
-$events = CleverEvent::all(array('ending_before' => 'last', 'limit' => 1));
+?>
 ```
 
-To cursor through larger data sets, capture the last ID of an object in a set, and use it as the `starting_after` value for a subsequent request.
+## Updating the Library
 
-```php
-$schools = CleverSchool::all(array('limit' => 1)); # page "one"
-$last_school = $schools[count($schools)-1];
-do {
-  # retrieve page "two" and beyond
-  $more_schools = CleverSchool::all(array('limit' => 1, 'starting_after' => $last_school->id));
-  if(count($more_schools) > 0) {
-    $schools = array_merge($schools, $more_schools);
-    $last_school = $more_schools[count($more_schools)-1];
-  } else {
-    $last_school = NULL;
-  }
-} while($last_school);
-print_r($schools); # Print all the schools retrieved.
-```
+1. Git clone swagger-codegen (https://github.com/swagger-api/swagger-codegen)
 
-To cursor to a previous page, use the first ID of each set as the `ending_before` value.
+2. Git clone Clever's swagger-api repo (https://github.com/Clever/swagger-api)
 
-Unfortunately, paginating via relative links is not yet supported by clever-php.
+3. Are you removing some old models? Consider removing existing files from `docs/` and `lib/` and `/test` before regenerating files
```

Comment:
should we just delete these files as part of the generation?

Reply:
Done